letterboxd/build.gradle.kts

```diff
@@ -1,35 +1,50 @@
 @Suppress("DSL_SCOPE_VIOLATION") // TODO: Remove once KTIJ-19369 is fixed
 plugins {
-    alias(libs.plugins.build.config)
-    alias(libs.plugins.org.jetbrains.kotlin.jvm)
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.kotlin.android)
     alias(libs.plugins.ksp)
     kotlin("kapt")
 }
 
-buildConfig {
```

Comment:
Didn't need this anymore since android library already has support for build config.